Improvements to installation instruction

mattias-p opened the pull request:
I made some improvements to the installation instruction for Debian while testing.

sandoche2k commented:
@mattias-p we could approve this only if we test it on Debian. So the approval may take time.

mattias-p commented:
@sandoche2k I hope this doesn't invalidate too much of your testing.

matsduf commented on docs/Installation.md:
````diff
@@ -544,7 +544,7 @@ you should be able to use the
 
 ```sh
-curl -s -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","method":"version_info","id":"1"}' http://localhost:5000/ && echo
+wget -q -O - --header "Content-Type: application/json" --post-data '{"jsonrpc":"2.0","method":"version_info","id":"1"}' http://localhost:5000/ && echo
````
Please give a strong argument to change from curl to wget at this late stage. I am against the change for v2019.1.

matsduf commented:
The commit comment is "Use wget instead of curl on Debian", but the change is on OSs.

sandoche2k commented:
WGET vs CURL
https://daniel.haxx.se/docs/curl-vs-wget.html. 
I do support @matsduf that we may discuss it for next version

mattias-p commented:
I don't think curl vs wget is a big deal in this case. Feature-wise both meet our needs. Wget comes pre-installed, at least on some platforms, which is convenient. Removed the commit to sidestep the argument.

matsduf commented:
@mattias-p, the change in installation of dependencies doe not look like an editorial change. Is it? Can you please describe?

mattias-p commented:
@matsduf Well, as I described in the commit message, I've added some forgotten dependencies,
shifted to install some dependencies from APT instead of CPAN, and
removed some transitive and/or unused dependencies.

matsduf commented:
Sorry, the Github hides long commit messages and I did not pay attention to the dots. I think it is more visible when repeated in the description of the PR.

mattias-p commented:
Thanks, @matsduf.

Waiting to merge until it's been tested on Ubuntu.

sandoche2k commented:
@mattias-p merge?
